pkg/instancegroups/instancegroups.go

```diff
@@ -262,7 +276,6 @@ func (r *RollingUpdateInstanceGroup) tryValidateCluster(rollingUpdateData *Rolli
 		klog.Infof("Cluster did not pass validation, will try again in %q until duration %q expires: %s.", tickDuration, duration, strings.Join(messages, ", "))
 		return false
 	} else {
-		klog.Info("Cluster validated.")
```

Comment:
It might be worth keeping this, or at least letting the user know about the first successful validation in the duration.